### PR TITLE
Update svg-icons and less files

### PR DESCRIPTION
**Pull request by ginold**
update SVG icons with the new waypoint type icons and separate into less files into activities and waypoint types.

related to #390

### CODE REVIEW

**Comment by asaunier** [comment]
Except the conflicts fixing, are there still work to do on this PR?

**Comment by asaunier** [comment]
Any SVG icon is missing for "image" documents.

**Comment by asaunier** [comment]
@ginold What's the status of this PR?

**Comment by ginold** [comment]
ready to be reviewed!

File: c2corg_ui/static/js/map/map.js [hunk 1]
```diff
@@ -404,14 +404,28 @@ app.MapController.prototype.createStyleFunction_ = function() {
...
   }
   var id = /** @type {number} */ (feature.get('documentId'));
   var highlight = /** @type {boolean} */ (!!feature.get('highlight'));
   var scale = highlight ? 1 : 0.5;
-  var imgSize = highlight ? 32 : 16;
+
+  if (module === 'waypoints') {
+    imgSize = highlight ? 40 : 24;
+    path = '/documents/waypoints/' + type + '.svg';
+  } else if (module === 'images') {
```
Comment (asaunier) [inline c2corg_ui/static/js/map/map.js]: Do you really need to make a special case for images since it's the same path and size than the default?

**Comment by asaunier** [comment]
Nice work! I have only spotted a strange piece of code else looks good. Please address my comment and merge if Travis is OK and no one objects in the meantime.

**Pull request merged**